### ISSUE TITLE
Steam In-Home Streaming from Windows to Linux causes grey/green screen

### ISSUE THREAD

**Issue by Pobega** [ValveSoftware/steam-for-linux#6066]
#### Your system information

* Steam client version (build number or date): Jan 31, 2019 (1548979876)
* Distribution (e.g. Ubuntu): Fedora 29
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes
* Hardware: Intel HD 5500
* Software: Mesa 18.3.1

Host Info:

* Steam client version (build number or date): Jan 31, 2019
* Distribution: Windows 10
* Opted into Steam client beta? Yes
* Hardware: AMD Vega 64

#### Please describe your issue in as much detail as possible:
As the title says, when I use In-Home streaming I get a very grey screen or a full green screen when trying to stream from my Windows PC. Streaming works fine on my Link and Link App, so the host isn't the issue.

Keep in mind this is using the Steam Flatpak, but native gives me the same result.

What's strange is that when I use the "Display Performance Information" option, it seems to work perfectly fine. But I'd prefer not to play with all of that information cluttering my screen.

I've tried all combinations of disabling/enabling hardware decoding on the host and client, but the only thing that gets it to work at all is enabling the performance information overlay.

Note: Start+Y on the controller mostly disables the HUD which is nice for playing, but still doesn't solve the initial issue.

#### Steps for reproducing this issue:

1. Use a Windows host to stream a game to a Linux client
2. Green screen

**Comment by kisak-valve** [comment]
Hello @Pobega, please give #5337 a read and compare it to your experience. Like that issue, are you able to take screenshots of those games without IHS involved?

**Comment by Pobega** [comment]
Using the system screenshot tool? Here's a screenshot of it at the title screen with and without the debug overlay.

Note: I couldn't recreate the green screen, but it seems to sometime be green and sometimes be some sort of weird grey overlay.

![screenshot from 2019-02-01 11-20-22](https://user-images.githubusercontent.com/474525/52135356-9f476080-2613-11e9-88b8-f266def465dc.png)
![screenshot from 2019-02-01 11-21-12](https://user-images.githubusercontent.com/474525/52135363-a4a4ab00-2613-11e9-85f7-4a1ed28b78e4.png)

**Comment by mwpow3ll** [comment]
> Hello @Pobega, please give #5337 a read and compare it to your experience. Like that issue, are you able to take screenshots of those games without IHS involved?

I was the original reporter of #5337.

The new issue that @Pobega is having isn't limited to UE4 engines or Windows->Linux. I am streaming from Linux->Linux and I see the issue in all my games: native and non (Proton) with various engines.

- Stellaris (native client and proprietary paradox engine): Suffers from Green screen; hear audio in background
- Rimworld (native client and Unity engine): Suffers from inverted colors; hear audio in background
- Divine Divinity (proton): Suffers from inverted colors
- Helium Rain (native client and UE4 engine): Suffers from green screen; hear audio in background
- Hammerwatch (native client): Suffers from green screen; hear audio in background
- Firewatch (native client): Suffers from inverted colors; hear audio in background

To recap this is a **MAJOR** Linux streaming regression that affects all games from what I can tell. 

Might be a Fedora compatibility issue, given @Pobega and I are running Fedora 29

Steam client version (build number or date): Jan 31, 2019 (1548979876)
Distribution (e.g. Ubuntu): Fedora 29
Opted into Steam client beta?: No
Have you checked for system updates?: Yes
Hardware: Nvidia GTX 1080 TI
Software: Mesa 18.2.8 / Nvidia 415.27

**Comment by romanovzky** [comment]
I have the exact same issue. Steam updated today (I was not enrolled in beta) and the problem emerged. Host is windows 10, client is ubuntu 18.04. Changing various options does not change the result, except for the performance monitor, as said above.

From the forums this seems to be a generalised issue: https://steamcommunity.com/groups/homestream/discussions/0/1743357605566991061/

**Comment by Melinoe** [comment]
Same issue. Just wanted to add that same happens with host as a linux system (tried both windows and linux on host, client is laptop with only linux). 

Also because this happened after self updater updated steam I tried reverting to an earlier steam version still in my package managers cache (from steam 1.0.0.59-1 to 1.0.0.56-1), however that did not have any effect on the issue.

**Comment by duck511** [comment]
Same issue here, streaming from Win 10 to Linux (elementary), the image is mostly green and badly distorted. Broke immediately after the client update on 1 Feb 2019.

Processor Information:
    CPU Vendor:  AuthenticAMD
    CPU Brand:  AMD A10-7300 Radeon R6, 10 Compute Cores 4C+6G 
    CPU Family:  0x15
    CPU Model:  0x30
    CPU Stepping:  0x1
    CPU Type:  0x0
    Speed:  1900 Mhz
    4 logical processors
    4 physical processors
    HyperThreading:  Unsupported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Supported
    SSE41:  Supported
    SSE42:  Supported
    AES:  Supported
    AVX:  Supported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Unsupported

Operating System Version:
    elementary OS 5.0 Juno (64 bit)
    Kernel Name:  Linux
    Kernel Version:  4.15.0-45-generic
    X Server Vendor:  The X.Org Foundation
    X Server Release:  11906000
    X Window Manager:  Mutter(Gala)
    Steam Runtime Version:  steam-runtime-beta-release_2019-01-10

Video Card:
    Driver:  X.Org AMD KAVERI (DRM 2.50.0 / 4.15.0-45-generic, LLVM 6.0.0)
    Driver Version:  3.0 Mesa 18.0.5
    OpenGL Version: 3.0
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 60 Hz
    VendorID:  0x1002
    DeviceID:  0x130a
    Revision Not Detected
    Number of Monitors:  1
    Number of Logical Video Cards:  1
    Primary Display Resolution:  1920 x 1080
    Desktop Resolution: 1920 x 1080
    Primary Display Size: 13.54" x 7.60" (15.51" diag)
                                            34.4cm x 19.3cm (39.4cm diag)
    Primary VRAM: 1024 MB

Sound card:
    Audio device: ATI R6xx HDMI

Memory:
    RAM:  6899 Mb

**Comment by LenPayne** [comment]
I identified this in the beta version and rolled back to non-beta. I regret not chiming in earlier.

Tested this afternoon. I loaded a streaming game before applying today's update and the game appeared fine. I applied the new Steam client update, loaded the exact same game, and was presented with a pure green screen. (Usually the colours have been inverted. Green was an interesting change.)

I play almost exclusively on my lower-end Linux laptop streaming from my Win10 gaming rig.

```
Steam client application
    Built: Jan 31 2019, at 23:11:14
    Steam API: v018
    Steam package versions: 1548979876

Computer Information:
    Manufacturer:  Unknown
    Model:  Unknown
    Form Factor: Laptop
    No Touch Input Detected

Processor Information:
    CPU Vendor:  GenuineIntel
    CPU Brand:  Intel(R) Core(TM) i5-5300U CPU @ 2.30GHz
    CPU Family:  0x6
    CPU Model:  0x3d
    CPU Stepping:  0x4
    CPU Type:  0x0
    Speed:  2900 Mhz
    4 logical processors
    2 physical processors
    HyperThreading:  Supported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Supported
    AES:  Supported
    AVX:  Supported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Unsupported

Operating System Version:
    Ubuntu 18.04.1 LTS (64 bit)
    Kernel Name:  Linux
    Kernel Version:  4.15.0-44-generic
    X Server Vendor:  The X.Org Foundation
    X Server Release:  11906000
    X Window Manager:  GNOME Shell
    Steam Runtime Version:  steam-runtime-beta-release_2019-01-10

Video Card:
    Driver:  Intel Open Source Technology Center Mesa DRI Intel(R) HD Graphics 5500 (Broadwell GT2) x86/MMX/SSE2
    Driver Version:  3.0 Mesa 19.0.0-devel - padoka PPA
    OpenGL Version: 3.0
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 60 Hz
    VendorID:  0x8086
    DeviceID:  0x1616
    Revision Not Detected
    Number of Monitors:  1
    Number of Logical Video Cards:  1
    Primary Display Resolution:  1920 x 1080
    Desktop Resolution: 1920 x 1080
    Primary Display Size: 12.20" x 6.69" (13.90" diag)
                                            31.0cm x 17.0cm (35.3cm diag)
    Primary VRAM Not Detected

Sound card:
    Audio device: Intel Broadwell HDMI

Memory:
    RAM:  7661 Mb

Miscellaneous:
    UI Language:  English
    LANG:  en_CA.UTF-8
    Total Hard Disk Space Available:  167983 Mb
    Largest Free Hard Disk Block:  58093 Mb
    VR Headset: None detected```

**Comment by BenMueller** [comment]
Same thing affecting me. Host is AMD 580 and client is NVIDIA Quadro M2000M, Linux on both hosts. Just anecdotal, but the host was in the beta and the client wasn't; I started having issues when the client updated earlier today. Client-side bug?

**Comment by RNanclares** [comment]
Same problem here. It  was working perfectly fine until last update. Host Windows 10 - Client Linux (Manjaro Gnome - Intel Graphics).

**Comment by Betriebsrat** [comment]
Exact same issue after update for me too
Windows 10 Host
Linux Client

Displaying performance turning it back to normal tho

**Comment by stijnghesquiere** [comment]
Same issue here. 
Since yesterday, bad quality streaming, washed out. It disappears if I turn on performance statistics. 
Windows 10 host, NVIDIA GPU; I7
Ubuntu 18.04 client, intel  graphics 655, i3
The issue is with ALL games.

**Comment by poofyteddy** [comment]
Same issue, I had it on steam beta and needed to rollback to mainstream in order to fix it.
With the last update, it seam that the issue have been brought to main ...
Windows 8.1 host, Nvidia 1080Ti, i5
Linux Fedora 29 client Nvidia 1080, I7

**Comment by chriscamacho** [comment]
streaming from Linux to Linux also shows this issue

**Comment by miruoy** [comment]
Same issue after latest update. Issue emerges on 2 nodes. Switching between Beta and Stable release does not make a difference anymore.

Clent Node 1 openSUSe Leap  15.0 Radeon graphics
Client Node 2 openSUSe Leap  15.0 Intel Atom Graphics
Server Node Win 10 AMD R9 390 19.1.1

Enabling "Display Performance Information" confirmed as workaround.

**Comment by Urworstnit3m3r** [comment]
**Your system information**

    Steam client version (build number or date): Jan 31, 2019 (1548979876)
    Distribution (e.g. Ubuntu): Manjaro (64 bit)
    Opted into Steam client beta?: No
    Have you checked for system updates?: Yes
    Hardware: i7 6500U, Intel HD 520
    Software: 3.0 Mesa 18.3.2

**Host Info:**

    Steam client version (build number or date): Jan 31, 2019
    Distribution: Windows 10
    Opted into Steam client beta? No
    Hardware: 4790k, GTX 980

**Please describe your issue in as much detail as possible:**

When I use In-Home streaming I get washed out colors when trying to stream from my Windows PC. 

![washed_out_colors](https://user-images.githubusercontent.com/37315608/52166666-ccf4de00-26d5-11e9-8594-0a2f669e56b3.png)

As others have said when you use the "Display Performance Information" option, it works perfectly fine. 

![correct_colors_with_display_perf](https://user-images.githubusercontent.com/37315608/52166667-d2522880-26d5-11e9-8ce0-b0a89f9717b3.png)

**Steps for reproducing this issue:**

    Use a Windows host to stream a game to a Linux client

**Comment by LenPayne** [comment]
Someone on the Steam Forums thread posted that they're experiencing this with the macOS Streaming Client too.

**Comment by frnknstn** [comment]
I am getting this issue as well, since yesterday. It's like YCrCb data is being drawn to an RGB surface or something.

I am streaming from Windows 10 to Ubuntu 16.04.

**Comment by mostman1043** [comment]
I can confirm this happens on Mac clients too.

**Comment by eikenb** [comment]
I have 2 different systems set up for streaming, for my wife and myself. My wife's system is experiencing this and mine isn't. The hosts are almost exactly the same.

Hosts: Intel CPUs (i7-4790K), Nvidia 970GTX
The only difference between hosts is 1 has a Fit-Headless (max 1080p) and 1 has a Fit-Headless-GS (set to 1440p).
Both hosts have latest Nvidia drivers.

Clients are different.
Without problem: Debian stable (9.7), Lenovo 450s (Intel CPU, Intel 5500 GPU)
With problem: Ubuntu 18.04, AMD Ryzen 5 1600, Nvidia 1050 GPU (using latest drivers)

I tried with and without hardware acceleration on both. Updated video drivers on both hosts and affected client. Nothing helped.

**Comment by ryoko227** [comment]
Just made an account to add my comment as well.

Win10-64 -> Mint 18.3 KDE
Exactly as everyone else has described above.
I don't think a list of my hardware is relavent as this (IMHO) has been confirmed as a Steam software regression issue. My screens look the same as above, including the work around screenshots.

0) No issues streaming last night for literally years prior
1) Steam updated this morning (JST)
2) Tried to stream a game from the Win10 Host
3) Got the ultra grey somtimes green screen
4) Searched internet for options
5) Tried with "Display performance information" - works with no display issues

Thank you in advance for fixing this as quickly as possible ^_^v

**Comment by ronchaine** [comment]
Happens for me with Linux host -> Linux client as well.

```
Computer Information:
    Manufacturer:  Unknown
    Model:  Unknown
    Form Factor: Laptop
    No Touch Input Detected

Processor Information:
    CPU Vendor:  GenuineIntel
    CPU Brand:  Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
    CPU Family:  0x6
    CPU Model:  0x8e
    CPU Stepping:  0xa
    CPU Type:  0x0
    Speed:  4000 Mhz
    8 logical processors
    4 physical processors
    HyperThreading:  Supported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Supported
    AES:  Supported
    AVX:  Supported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Unsupported

Operating System Version:
    Linux 4.x (64 bit)
    Kernel Name:  Linux
    Kernel Version:  4.18.8-2-surface4
    X Server Vendor:  The X.Org Foundation
    X Server Release:  12001000
    X Window Manager:  i3
    Steam Runtime Version:  steam-runtime-beta-release_2019-01-10

Video Card:
    Driver:  Intel Open Source Technology Center Mesa DRI Intel(R) UHD Graphics 620 (Kabylake GT2) x86/MMX/SSE2
    Driver Version:  3.0 Mesa 18.3.2
    OpenGL Version: 3.0
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 59 Hz
    VendorID:  0x8086
    DeviceID:  0x5917
    Revision Not Detected
    Number of Monitors:  1
    Number of Logical Video Cards:  1
    Primary Display Resolution:  3000 x 2000
    Desktop Resolution: 3000 x 2000
    Primary Display Size: 10.63" x 7.09" (12.76" diag)
                                            27.0cm x 18.0cm (32.4cm diag)
    Primary VRAM Not Detected

Sound card:
    Audio device: Realtek ALC295

Memory:
    RAM:  15809 Mb

Miscellaneous:
    UI Language:  English
    LANG:  en_GB.UTF-8
    Total Hard Disk Space Available:  163722 Mb
    Largest Free Hard Disk Block:  136449 Mb
    VR Headset: None detected

Recent Failure Reports:
```

**Comment by saracoth** [comment]
Experiencing the same discoloration with Arch Linux streaming client (last full system update on 2019-01-26) and Windows 10 streaming host.

For as long as I can remember, my beta status has been "Not participating" on both systems. This issue was not present on Friday, but was present starting Saturday. The performance information workaround suggested by others does prevent the discoloration when enabled.

Advanced client options (Linux side):
- Automatic bandwidth limit
- Limit to display resolution
- Stereo speakers
- Enabled hardware decoding
- Enabled display performance information (enabled as a workaround suggested by others)

Advanced host options (Windows side):
- Disabled changing host resolution to match client
- Disabled dynamic capture resolution
- Disabled NVFBC capture
- Enabled hardware encoding on all listed graphics card types
- Automatic encoding thread count
- Disabled network prioritization

Linux client's system information:

```
Computer Information:
    Manufacturer:  Unknown
    Model:  Unknown
    Form Factor: Desktop
    No Touch Input Detected

Processor Information:
    CPU Vendor:  AuthenticAMD
    CPU Brand:  AMD A10-7850K Radeon R7, 12 Compute Cores 4C+8G
    CPU Family:  0x15
    CPU Model:  0x30
    CPU Stepping:  0x1
    CPU Type:  0x0
    Speed:  3700 Mhz
    4 logical processors
    4 physical processors
    HyperThreading:  Unsupported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Supported
    SSE41:  Supported
    SSE42:  Supported
    AES:  Supported
    AVX:  Supported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Unsupported

Operating System Version:
    Linux 4.x (64 bit)
    Kernel Name:  Linux
    Kernel Version:  4.20.4-arch1-1-ARCH
    X Server Vendor:  The X.Org Foundation
    X Server Release:  12003000
    X Window Manager:  Xfwm4
    Steam Runtime Version:  steam-runtime-beta-release_2019-01-10

Video Card:
    Driver:  X.Org AMD Radeon R9 200 Series (HAWAII, DRM 3.27.0, 4.20.4-arch1-1-ARCH, LLVM 7.0.0)
    Driver Version:  4.5 (Compatibility Profile) Mesa 18.3.1
    OpenGL Version: 4.5
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 60 Hz
    VendorID:  0x1002
    DeviceID:  0x67b1
    Revision Not Detected
    Number of Monitors:  1
    Number of Logical Video Cards:  2
    Primary Display Resolution:  1920 x 1080
    Desktop Resolution: 1920 x 1080
    Primary Display Size: 62.99" x 35.43" (72.24" diag)
                                            160.0cm x 90.0cm (183.5cm diag)
    Primary VRAM: 4096 MB

Sound card:
    Audio device: ATI R6xx HDMI

Memory:
    RAM:  14962 Mb

Miscellaneous:
    UI Language:  English
    LANG:  en_US_custom.UTF-8
    Total Hard Disk Space Available:  2081091 Mb
    Largest Free Hard Disk Block:  2019124 Mb
    VR Headset: None detected

Recent Failure Reports:
```

**Comment by Floumen** [comment]
Good day! I am writing to you with a request to help me with the home broadcast of Steam. I have three different computers, each of which is Windows 10 and Linux (1 PC) and Linux (2 PC). If you connect to the broadcast from a computer (Linux 1) to a computer (Linux 2), then there is a problem, as in the video. The same thing happens if you connect to the broadcast on Windows. If you connect to the broadcast from Windows to any Linux, then this problem is not observed. Everything worked until the last Steam update. Reinstalling (including manual removal of the Steam folder) on all three devices did not help.

The problem arose precisely after the last Steam update. I tried to arrange settings in Home Broadcast in a different order.

Specifications:
- Windows 10 Pro (clean install, only Steam and nVidia drivers are installed)
(Geforce GTX 285)
- Linux 1: Deepin 15.9.1 (based on Debian) nvidia-390.67 driver (driver has not changed) (Geforce GTX 650)
- Linux 2: Deepin 15.9.1 (based on Debian) nvidia-390.67 driver (driver has not changed)
(Geforce 940M)

Видео:
https://youtu.be/2wbYrnzwYJg

**Comment by jahrain** [comment]
I'm having  the same problem. Looks like the color format of the codec is mismatched between the client and host. Like as if one side is interpreting YUV colors as RGB or something.

Host: Windows 10 Pro 64bit - NVIDIA GTX 1080 (Driver 417.71)

Client: Ubuntu 16.04 x86_64 (Kernel 4.13.0-40)  - NVIDIA GTX 960 (Driver 415.27)

**Comment by aetimmes** [comment]
Same issue
Host: Win10 Education 64bit - NVidia GeForce 1060
Client: Fedora 29 (kernel 4.20.5-200.fc29.x86_64) - NVS 5400M

**Comment by clefru** [comment]
@kisak-valve It might be that the glxChooseFBConfig call in streaming_client gets the wrong FB config with the steam updated version. It might be worthwhile to see what glxChooseFBConfig returns when performance stats are enabled, and when they are not, and then add some attributes to that call to restrict the returned list to a compatible FB config.

**Comment by rdotts** [comment]
Same issue

Host: Win 10 Home 1809, 2x Nvidia GTX 780 SLI

Client: Kubuntu 18.10, AMD 5400k w/integrated Radeon 7450d

Connection: Wired Gigabit

Streaming worked great with the same basic setup 2-3 months ago

**Comment by vonmoltke** [comment]
The hardware decoding setting does not change anything for me.  Only
showing performance stats corrects the problem.

For reference:

Host:
Custom-build
Windows 10
AMD Ryzen 7 processor
Nvidia GTX960 graphics card

Client:
Dell XPS13 (9350)
Debian unstable (last updated on the evening (EST) of Friday 01FEB)

**Comment by geefr** [comment]
Same issue here

Host: Win 10 Home 1809, Nvidia GTX1060 (Running in VM, gpu passthrough from streaming client)

Client: Mint 19, Nvidia GTX1050. Runs host machine under KVM with vfio for pci passthrough

Connection: Virtual network bridge (Essentially wired, ~50Gbit)

Streaming worked perfectly in the past, now busted
Enabling the performance overlay corrects the colours. Noted with the overlay enabled it's using software decoding through avformat despite hardware decoding being enabled. Can't tell if hardware decoding is working without the overlay.

All hardware capture/decoding options are enabled in Windows, changing them makes no difference so likely on the decoding side.

**Comment by VADemon** [comment]
Bump. Looks like **chroma** channels are messed up but it occurs ONLY if the performance metrics are disabled.
Host:

- OS: Windows 7 SP1 x64

- GPU: RX 580

- Capture: Software only, 2 threads

Client:

- OS: Ubuntu

**Comment by Chrismettal** [comment]
Same issue. Glad it's not just me then.

## Client:
CPU: AMD Athlon 8 9600
Graphics: Card: iGPU of Athlon 8 9600
Desktop: KDE Plasma 
Distro: Antergos OS

## Host:
CPU: Ryzen 2700x
GPU: RTX 2080 Ti
Windows 10 Pro

All software is at the latest stable version dated 04.02.19
Resolution and aspect ratio of the Host did not seem to change if i set everything to lower my resolution to 1080p (Originalle 21:9 1440p UWQHD).  Gonna try that this evening. 

Tested all streaming settings other than the "Show performance metric" thingy. Thanks for the temporary fix idea.

**Comment by danginsburg** [comment]
We've got a fix for this.  The fix will be in the next Steam Beta update.

**Comment by ryoko227** [comment]
> We've got a fix for this. The fix will be in the next Steam Beta update.

So, safe to assume we need beta participation enabled, specifically on the client side? Or both host and client?

**Comment by danginsburg** [comment]
Should be just on the client and yes requiring beta participation. CC @slouken to correct me if I'm wrong.

**Comment by ajvengo** [comment]
Please backport this hotfix to main Linux Steam client. This is a major regression, which deserve fast hotfix merge into release code branch.

**Comment by Yeest** [comment]
Host PC Windows 10 -> Client Mac OSX 10.13 , same issue here. What seemed to have triggered it was prompting a stream of a game that was already open and running on the host.

**Comment by Houndie** [comment]
Just mentioning that I literally have this bug bookmarked and check it every day to see if the fix is ported yet.  The workaround is great, but having all the performance text on the screen really detracts from the experience, especially when playing HUD-heavy games.

**Comment by theit8514** [comment]
If you press F6 on the client it reduces the amount of text to two lines.

**Comment by AlexVegner** [comment]
Same issue: 
Client: OSX 10.14, Steam with last updates
Host Window 10, RTX 2080Ti,  Steam with last updates
Workaround doesn't work. 
Same issue for last stable and last beta steam builds

**Comment by hhirsch** [comment]
Same issue with Linux to Linux

**Comment by tliron** [comment]
Hey everyone, just so you know: when you use the performance text workaround you can press F6 to make it  smaller. A workaround to the workaround. :)

**Comment by nioc** [comment]
Same here with following setup:

Client:
- Steam client version: Feb 2, 2019 (1549129917)
- Distribution: Ubuntu 18.04.1
- Hardware: Laptop Asus, NVIDIA Corporation GeForce GT 730M/PCIe/SSE2
- Software: 4.6.0 NVIDIA 410.93
- CPU: Intel(R) Core(TM) i7-4500U CPU @ 1.80GHz
- Opted into Steam client beta?: No
- Have you checked for system updates?: Yes

Host:
- Distribution: Windows 7
- Hardware: AMD Radeon HD 6800 Series
- Software: 15.301.1901.0
- CPU: Intel(R) Core(TM) i5-2500K CPU @ 3.30GHz

**Comment by HarlockSystem** [comment]
Same here
client :
 - Steam client version: Feb 2, 2019 (1549129917) 
 - Linux 4.19.18-1-lts "Arch Linux"
 - nvidia 1060 GTX 
   Driver Version: 415.27
   OpenGL Version: 4.6
  -Opted into Steam client beta?: No
  - Have you checked for system updates?: Yes

Host:
- windows 10 pro 1803
- nvidia geforce 960m
- Steam client version: Feb 2, 2019 (1549129917)

**Comment by LenPayne** [comment]
Latest beta version is now released. Thank you, this has been resolved.

**Comment by evilynux** [comment]
Oh, I was wondering whether or not it was fixed with the latest release. Great news then! Just didn't make it to the changelog. Thanks for the input @LenPayne !

**Comment by evilynux** [comment]
Confirmed, fixed for me as well with the beta release of Feb 8th. Thanks!

**Comment by teleportingtortoise** [comment]
Still an issue here even after changing both the client and host to the beta and restarting Steam, forcing a check for updates didn't help either. Windows 10 host with a nvidia 1050ti and Arch Linux client with a nvidia gt 730.

**Comment by kukiric** [comment]
Same here on Arch. Issue happens when neither version is on the beta, when only the client is on the beta, and when both systems are on the beta version of Steam. Disabling hardware encoding on the host, or decoding on the client also doesn't work, neither does disabling Nvidia NVFBC or setting the host to use the client's desktop resolution.

The only workaround is enabling "Display Performance Information" on the client.

Host computer: Windows 10 with an Nvidia GTX 970

Client computer: Arch Linux (just updated 15 minutes ago) with an Intel HD 530

**Comment by geefr** [comment]
Fixed for me, beta on both host and client, all hw encoding options including NVFBC enabled on both host and client.

**Comment by Venturina** [comment]
For me, enabling beta on client worked as well!

**Comment by miruoy** [comment]
Confirmed as resolved on Leap 15.

**Comment by teleportingtortoise** [comment]
Now resolved as of this morning for me as well.

**Comment by rgentz** [comment]
Thanks. I only needed the linux beta. Then windows host is still running the regular version

**Comment by 8BitCerberus** [comment]
Confirming it's fixed for me. Ubuntu 18.04 client, Windows 10 host, Steam beta client on both.

**Comment by vshynkar** [comment]
When this fix will be released in stable version?

**Comment by Ziris85** [comment]
Just throwing my 2c in here that, switching to the beta steam client on my linux computer (in my case, CentOS 7) and leaving the steam client on the Windows host on stable has resolved the color issues

**Comment by kukiric** [comment]
Just tried the latest Linux beta again, and it works correctly with hardware decoding enabled. Dunno why it didn't a few days ago (maybe the beta didn't reach the my local CDN edge), but it looks good now.

**Comment by sonofapharmacist** [comment]
I have a solution! The nvidia geforce experience game overlay was the culprit for me. on the server device i disabled the in-game overlay and it worked great.

**Comment by bskendig** [comment]
Disabling the Nvidia Geforce Experience in-game overlay did _not_ solve the problem for me. I still have a greyed-out display when streaming from Windows 10 to macOS Mojave (latest patches, latest non-beta Steam client on each, GeForce 970 with latest drivers).

**Comment by bskendig** [comment]
Mac Mac Steam client updated itself to a new version, and it looks like that solved the problem for me.

**Comment by kisak-valve** [comment]
The fix for this has reached Steam's stable release.
